### DISCUSSION TITLE
How to change the colors?

### DISCUSSION THREAD

**Discussion by kintrupf** [danilrez/better-plsql#2] [Q&A] [answered]
How do I change the colors for the syntax items? I did not find a way to change them to my preference anywhere.

**Comment by danilrez** [comment, marked as answer]
Hi! In "Settings.json", if you wrote this, everything should work fine:
"files.associations": {
    "*.pck": "plsql-ccm",
    "*.sql": "plsql-ccm",
}

**Reply by sandra12-23** [reply]
Are the colors set in stone or can I set my own colors somewhere?

**Reply by danilrez** [reply]
This extension offers enhanced highlighting for the PL/SQL language compared to the default SQL highlighting provided by the Language Mode menu. In the default SQL highlighting, all keywords are grouped together, whereas this extension separates them into appropriate groups, resulting in improved code readability.

Regarding colors, they depend on the theme you have selected for Visual Studio Code. Personally, I use my own themes. You can find the link to the marketplace where you can download my themes here: _[Marketplace link](https://marketplace.visualstudio.com/items?itemName=CrappyCodeMaker.crappycode-theme)_

**Reply by sandra12-23** [reply]
Thanks
